src/fountain_source.cc

```diff
@@ -18,7 +18,7 @@
 namespace Gem {
 namespace Particle {
 
-glm::f32vec3 const FountainSource::DEFAULT_SPEED = { 0.0f,2.0f,0.0f };
```

Comment:
This should be some sort of macro or something that lets every particle have a random behavior.